### DISCUSSION TITLE
Orphan Files with 4GB and caracter ^ in the names.

### DISCUSSION THREAD

**Discussion by patrickdalla** [sepinf-inc/IPED#537] [Q&A]
I've processed FAT filesystem, and it resulted in many files with "^^^^^^^^.^^^" and 4GB in size.
How does IPED/SleuthKit recover these orphan files?

**Comment by patrickdalla** [comment]
It seems that these are false positive orphan files (there never had been such file). So sleuth recover non-printable characters in the name that are substituted with '^' and 4GB in the size field.

**Reply by lfcnassif** [reply]
Yes, those are usually garbage recovered by sleuthkit. There is the **minOrphanSizeToIgnore** option in **AdvancedConfig.txt** file to discard those files if needed. If sleuthkit recovers 1K files with 4GB size (pointing to the same unalloc areas), your thumbdrive will be processed like a 4TB disk, so that option should help this scenario.
